Cargo.toml

```diff
@@ -75,6 +74,7 @@ members = [
     "crates/transaction-emitter-lib",
     "ecosystem/node-checker",
     "ecosystem/node-checker/fn-check-client",
+    "ecosystem/sf-indexer/firehose-stream",
```

Comment:
Wondering why you have to do this

Reply:
This was missing in the Cargo.toml, so it was not part of the workspace, but still uses Move references.